package-deploy.yaml

```diff
@@ -30,8 +30,6 @@ agents:
     dockerFilePath: docker/build/aptly.Dockerfile
     image: algorand/aptly
     version: scripts/configure_dev-deps.sh
-    buildArgs:
-      - GOLANG_VERSION=`./scripts/get_golang_version.sh`
```

Comment:
this appears to be still used in docker.ubuntu.Dockerfile ?

Reply:
Yes, but this is for the aptly container. We were doing a golang install for no reason, so I removed it.